project/1.18/assets/tinkers-construct/tconstruct/lang/zh_cn.json

```
@@ -1,2660 +1,2660 @@
+  "modifier.tconstruct.flamewake.flavor": "这可真行",
+  "modifier.tconstruct.flamewake.description": "在你的身后留下一条长长的火痕",
+
+
+  "_comment": "Abilities",
+  "modifier.tconstruct.unbreakable": "不毁",
+  "modifier.tconstruct.unbreakable.flavor": "冷冻封存！",
+  "modifier.tconstruct.unbreakable.description": "常规使用时，工具不再消耗耐久值",
+  "recipe.tconstruct.modifier.unbreakable_requirements": "“不毁”需要“下界合金”和“加固 V”作为作为前置",
+  "modifier.tconstruct.dual_wielding": "双持",
+  "modifier.tconstruct.dual_wielding.flavor": "二刀流！",
+  "modifier.tconstruct.dual_wielding.description": "主手可以双持的情况下，可从副手进行攻击。双持工具的属性将会降低",
+  "recipe.tconstruct.modifier.two_handed_dual_wielding": "双持工具需要“副手持握”强化才能使用此强化。",
+
+  "_comment": "Loot",
+  "modifier.tconstruct.silky": "丝触",
+  "modifier.tconstruct.silky.flavor": "纵享丝滑！",
+  "modifier.tconstruct.silky.description": "工具所开采的方块将掉落其自身",
+  "modifier.tconstruct.autosmelt": "自动烧炼",
+  "modifier.tconstruct.autosmelt.flavor": "下界之火蕴于工具之中！",
+  "modifier.tconstruct.autosmelt.description": "自动烧炼工具所采掘的方块",
+  "modifier.tconstruct.luck": "幸运",
+  "modifier.tconstruct.luck.1": "幸运",
+  "modifier.tconstruct.luck.2": "幸运+",
+  "modifier.tconstruct.luck.3": "幸运++",
+  "modifier.tconstruct.luck.flavor": "鸿运当头！",
+  "modifier.tconstruct.luck.description": "采掘与击杀生物时，能提升获得高品质战利品的几率！",
+  "recipe.tconstruct.modifier.luck.level_2": "“幸运+”的合成与前一级不同",
+  "recipe.tconstruct.modifier.luck.level_3": "“幸运++”的合成与前两级不同",
+  "modifier.tconstruct.fortune": "时运",
+  "modifier.tconstruct.fortune.flavor": "采矿幸运儿！",
+  "modifier.tconstruct.fortune.description": "采掘矿石时获得更多掉落物！",
+  "modifier.tconstruct.looting": "抢夺",
+  "modifier.tconstruct.looting.flavor": "战利幸运儿！",
+  "modifier.tconstruct.looting.description": "击杀生物后获得更多战利品！",
+  "recipe.tconstruct.modifier.looting.level_2": "“抢夺 II”的合成与前一级不同",
+  "recipe.tconstruct.modifier.looting.level_3": "“抢夺 III”的合成与前两级不同",
+
+  "_comment": "Expanders",
+  "modifier.tconstruct.expanded": "延展",
+  "modifier.tconstruct.expanded.flavor": "真的有在好好练腿哦！",
+  "modifier.tconstruct.expanded.description": "扩大工具的效用范围",
+  "modifier.tconstruct.reach": "长臂",
+  "modifier.tconstruct.reach.flavor": "心灵促动！",
+  "modifier.tconstruct.reach.description": "持握该工具时延伸触及距离，以便进行更远的采掘与攻击！",
+  "_comment": "Block transform",
+  "modifier.tconstruct.pathing": "铺路",
+  "modifier.tconstruct.pathing.flavor": "有路走遍天下！",
+  "modifier.tconstruct.pathing.description": "工具可制造土径",
+  "modifier.tconstruct.stripping": "剥皮",
+  "modifier.tconstruct.stripping.flavor": "树皮脱光光！",
+  "modifier.tconstruct.stripping.description": "工具可为木头去皮",
+  "modifier.tconstruct.tilling": "耕耘",
+  "modifier.tconstruct.tilling.flavor": "“犁”别后的重逢！",
+  "modifier.tconstruct.tilling.description": "工具可耕耘泥土",
+  "modifier.tconstruct.glowing": "荧光",
+  "modifier.tconstruct.glowing.flavor": "闪闪放光芒！",
+  "modifier.tconstruct.glowing.description": "持握者潜行时可放置荧光球，会消耗工具耐久",
+  "modifier.tconstruct.exchanging": "置换",
+  "modifier.tconstruct.exchanging.flavor": "!了转反",

```

```suggestion
  "modifier.tconstruct.exchanging.flavor": "！了转反",
```
以及搞笑版
```suggestion
  "modifier.tconstruct.exchanging.flavor": "¡了转反",
```
如果是排版原因就不改了